package.json

```
@@ -92,8 +92,9 @@
         "serve": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run serve $npm_config_open;}; f",
         "start": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run start ;}; f",
         "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && npm run test:eslint-all && npm run test:jest ;}; f",
+        "test:codecov": "jest --all --maxWorkers=4",
         "test:stylelint": "stylelint \"./packages/*/src/**/*.s(a|c)ss\"",
-        "test:jest": "jest --all --maxWorkers=2",
+        "test:jest": "jest --all --maxWorkers=8",
```

please use shell variable instead of having two different scripts for the same functionality
```suggestion
        "test:jest": "jest --all --maxWorkers=${1:8}",
```